reporter/aspec/src/main/java/org/itsallcode/openfasttrace/report/aspec/DelegatingXMLStreamWriter.java

```diff
@@ -0,0 +1,233 @@
+package org.itsallcode.openfasttrace.report.aspec;
```

Comment:
Copied from specobject report. @redcatbear should we create a `xml-util` submodule?

Reply:
Moved to new module exporter/common